menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -222,8 +227,19 @@ sap.ui.controller("components.schema.schemaMain", {
 			// nav back to info
 			this.byId("schemaIconTabBar").setSelectedKey("info");
 		}
-
-	}
+	} ,
+
+  validateSchemaFileUplaod : function () {
+    this.resetNewSchemaValueState();
+    var isOk = true;
```

Comment:
Let's use `let` instead of `var`. This isn't something we're doing everywhere yet, but we should try to move in that direction. The reason we are opting for `let` over `var` is that `let` is bound to the scope of the block it's in, while `var` isn't and could have unexpected consequences. [More info here.](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable-in-jav)

Reply:
let me apply the boy scout principle and do this for all the controllers.

Reply:
I don't think that's necessary, let's not expand the scope of this task unnecessarily. Just where you touch the code.

Reply:
ok got it.

Reply:
used let, Resolved

Reply:
> let

Reply:
resolved